ion-java-cli/pom.xml

```diff
@@ -0,0 +1,121 @@
+<!--
+  ~ Copyright 2007 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License").
+  ~ You may not use this file except in compliance with the License.
+  ~ A copy of the License is located at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ or in the "license" file accompanying this file. This file is distributed
+  ~ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+  ~ express or implied. See the License for the specific language governing
+  ~ permissions and limitations under the License.
+  -->
+
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.amazon.ion</groupId>
+  <artifactId>ion-java-cli</artifactId>
+  <version>1.0</version>
+  <packaging>jar</packaging>
+
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>
+    A Java CLI used for ion-test-driver
```

Comment:
I suggest: "A CLI that implements the standard interface defined by ion-test-driver."

Reply:
Changed.